src/gt4py/next/program_processors/runners/dace_fieldview/gtir_sdfg.py

```diff
@@ -274,14 +316,28 @@ def make_temps(field: gtir_builtin_translators.Field) -> gtir_builtin_translator
         temp_result = gtx_utils.tree_map(make_temps)(result)
         return list(gtx_utils.flatten_nested_tuple((temp_result,)))
 
-    def _add_sdfg_params(self, sdfg: dace.SDFG, node_params: Sequence[gtir.Sym]) -> list[str]:
-        """Helper function to add storage for node parameters and connectivity tables."""
+    def _add_sdfg_params(
```

Comment:
Why is this function not using `_collect_symbols_in_domain_expressions()` which has a very similar description purpose?

Probably you should rephrase the description to something like:
"GT4Py `Fields` arguments will be translated to `dace.data.Array` (We should make a PR that makes `Array` accessible as `dace.Array`). By default GT4Py Scalars arguments will be translated to `dace.data.Scalar` arguments, except the they are listed in `symbol_arguments`/`symbolic_arguments`, in which case they will be symbols inside the SDFG."

Reply:
Done.